test/integration/scenarios/network/peers_blacklist.js

```diff
@@ -15,37 +15,28 @@
 'use strict';
 
 const fs = require('fs');
-const childProcess = require('child_process');
 const Peer = require('../../../../logic/peer');
 const utils = require('../../utils');
-const blockchainReady = require('../../../common/utils/wait_for').blockchainReady;
-
-const totalPeers = 10;
-// Each peer connected to 9 other pairs and have 2 connection for bi-directional communication
-const expectedOutgoingConnections = (totalPeers - 1) * totalPeers * 2;
-const expectedConnectionsAfterBlacklisting = (totalPeers - 2) * (totalPeers - 1) * 2;
-
-module.exports = params => {
-	describe('Peer Blacklisted', () => {
-		const getAllPeers = sockets => {
-			return Promise.all(
-				sockets.map(socket => {
-					if (socket.state === 'open') {
-						return socket.call('list', {});
-					}
-				})
-			);
-		};
+const blockchainReady = require('../../../common/utils/wait_for')
+	.blockchainReady;
+const common = require('../common');
+
+module.exports = function(configurations, networkParameters) {
+	// Full mesh network with 2 connection for bi-directional communication without the blacklisted peer
+	const EXPECTED_OUTOGING_CONNECTIONS_AFTER_BLACKLISTING =
```

Comment:
That doesn't make sense, as we're not performing strict test. It's an estimation.

Reply:
I will open another issue to perform strict controls instead of `<=`.